### ISSUE TITLE
Building OpenSim GUI fails at the make PrepareInstaller step.

### ISSUE THREAD

**Issue by AlbertoCasasOrtiz** [opensim-org/opensim-gui#1399]
Building OpenSim GUI is failing at the `make PrepareInstaller` step.

### Steps to reproduce

Try to build OpenSim GUI, either in [CI](https://github.com/opensim-org/opensim-gui/actions), or in local following the [Build Instructions](https://github.com/opensim-org/opensim-gui/wiki/Build-Instructions).

### Expected result

The build process finishes normally.

### Actual result

The following errors are thrown, and the build fails:

```
    [mkdir] Created dir: /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/build/classes
 [nb-javac] Compiling 855 source files to /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/build/classes
 [nb-javac] warning: [options] bootstrap class path not set in conjunction with -source 1.6
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:293: error: cannot find symbol
 [nb-javac]     super(opensimCommonJNI.PropertyString_SWIGUpcast(cPtr), cMemoryOwn);
 [nb-javac]                           ^
 [nb-javac]   symbol:   method PropertyString_SWIGUpcast(long)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:318: error: cannot find symbol
 [nb-javac]     long cPtr = opensimCommonJNI.PropertyString_clone(swigCPtr, this);
 [nb-javac]                                 ^
 [nb-javac]   symbol:   method PropertyString_clone(long,PropertyString)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:326: error: cannot find symbol
 [nb-javac]     return opensimCommonJNI.PropertyString_getTypeName(swigCPtr, this);
 [nb-javac]                            ^
 [nb-javac]   symbol:   method PropertyString_getTypeName(long,PropertyString)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:345: error: cannot find symbol
 [nb-javac]     opensimCommonJNI.PropertyString_setValue__SWIG_0(swigCPtr, this, i, value);
 [nb-javac]                     ^
 [nb-javac]   symbol:   method PropertyString_setValue__SWIG_0(long,PropertyString,int,String)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:354: error: cannot find symbol
 [nb-javac]     opensimCommonJNI.PropertyString_setValue__SWIG_1(swigCPtr, this, value);
 [nb-javac]                     ^
 [nb-javac]   symbol:   method PropertyString_setValue__SWIG_1(long,PropertyString,String)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:364: error: cannot find symbol
 [nb-javac]     return opensimCommonJNI.PropertyString_getValue__SWIG_0(swigCPtr, this, index);
 [nb-javac]                            ^
 [nb-javac]   symbol:   method PropertyString_getValue__SWIG_0(long,PropertyString,int)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:374: error: cannot find symbol
 [nb-javac]     return opensimCommonJNI.PropertyString_getValue__SWIG_1(swigCPtr, this);
 [nb-javac]                            ^
 [nb-javac]   symbol:   method PropertyString_getValue__SWIG_1(long,PropertyString)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:384: error: cannot find symbol
 [nb-javac]     return new SWIGTYPE_p_std__string(opensimCommonJNI.PropertyString_updValue__SWIG_0(swigCPtr, this, index), false);
 [nb-javac]                                                       ^
 [nb-javac]   symbol:   method PropertyString_updValue__SWIG_0(long,PropertyString,int)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:394: error: cannot find symbol
 [nb-javac]     return new SWIGTYPE_p_std__string(opensimCommonJNI.PropertyString_updValue__SWIG_1(swigCPtr, this), false);
 [nb-javac]                                                       ^
 [nb-javac]   symbol:   method PropertyString_updValue__SWIG_1(long,PropertyString)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:403: error: cannot find symbol
 [nb-javac]     return opensimCommonJNI.PropertyString_appendValue(swigCPtr, this, value);
 [nb-javac]                            ^
 [nb-javac]   symbol:   method PropertyString_appendValue(long,PropertyString,String)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:413: error: cannot find symbol
 [nb-javac]     return opensimCommonJNI.PropertyString_adoptAndAppendValue(swigCPtr, this, SWIGTYPE_p_std__string.getCPtr(value));
 [nb-javac]                            ^
 [nb-javac]   symbol:   method PropertyString_adoptAndAppendValue(long,PropertyString,long)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:423: error: cannot find symbol
 [nb-javac]     return opensimCommonJNI.PropertyString_findIndex(swigCPtr, this, value);
 [nb-javac]                            ^
 [nb-javac]   symbol:   method PropertyString_findIndex(long,PropertyString,String)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:433: error: cannot find symbol
 [nb-javac]     return opensimCommonJNI.PropertyString_findIndexForName(swigCPtr, this, SWIGTYPE_p_SimTK__String.getCPtr(name));
 [nb-javac]                            ^
 [nb-javac]   symbol:   method PropertyString_findIndexForName(long,PropertyString,long)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:444: error: cannot find symbol
 [nb-javac]     return opensimCommonJNI.PropertyString_isA(AbstractProperty.getCPtr(prop), prop);
 [nb-javac]                            ^
 [nb-javac]   symbol:   method PropertyString_isA(long,AbstractProperty)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:454: error: cannot find symbol
 [nb-javac]     return new PropertyString(opensimCommonJNI.PropertyString_getAs(AbstractProperty.getCPtr(prop), prop), false);
 [nb-javac]                                               ^
 [nb-javac]   symbol:   method PropertyString_getAs(long,AbstractProperty)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] /Users/runner/work/opensim-gui/opensim-gui/Gui/opensim/modeling/src/org/opensim/modeling/PropertyString.java:464: error: cannot find symbol
 [nb-javac]     return new PropertyString(opensimCommonJNI.PropertyString_updAs(AbstractProperty.getCPtr(prop), prop), false);
 [nb-javac]                                               ^
 [nb-javac]   symbol:   method PropertyString_updAs(long,AbstractProperty)
 [nb-javac]   location: class opensimCommonJNI
 [nb-javac] 16 errors
 [nb-javac] 1 warning

BUILD FAILED
/Applications/NetBeans/Apache NetBeans 12.3.app/Contents/Resources/NetBeans/netbeans/harness/suite.xml:163: The following error occurred while executing this line:
/Applications/NetBeans/Apache NetBeans 12.3.app/Contents/Resources/NetBeans/netbeans/harness/common.xml:206: Compile failed; see the compiler error output for details.

Total time: 12 seconds
make[3]: *** [Gui/opensim/CMakeFiles/PrepareInstaller] Error 1
make[2]: *** [Gui/opensim/CMakeFiles/PrepareInstaller.dir/all] Error 2
make[1]: *** [Gui/opensim/CMakeFiles/PrepareInstaller.dir/rule] Error 2
make: *** [PrepareInstaller] Error 2
Error: Process completed with exit code 2.
```

### Environment and GUI version

- **Environments:** 1. Mac Big Sur, 2. GitHub Actions (Windows, Ubuntu and Mac)
- **GUI Version:** Latest from main branch.

**Comment by AlbertoCasasOrtiz** [comment]
A possible reason of this error and a solution is proposed in [this Pull Request](https://github.com/opensim-org/opensim-core/pull/3312) of the OpenSim Core repository.

**Comment by AlbertoCasasOrtiz** [comment]
Fixed in #1400
